src/packages.ts

```diff
@@ -248,7 +246,7 @@ export class InvalidPackage implements Package
     get workspace() { return undefined; }
     async debugConfiguration(): Promise<vscode.DebugConfiguration | undefined>
     {
-        throw new Error("Select a valid package before trying to create a debug configuration");
+        return Promise.reject(new Error("Select a valid package before trying to create a debug configuration"));
```

Comment:
What is the difference?

Reply:
A hope that the latter would not pollute the output with "un-handled promise errors" even if we actively handle it. It seems to be a problem with async/await.

Reply:
Did it make any difference?